### ISSUE TITLE
Dashboard with statistics for users/projects

### ISSUE THREAD

**Issue by gsteinLTU** [NetsBlox/NetsBlox#3326]
It would be nice to have some statistics about the users available to admins. Things such as users online, new accounts made, projects created, etc.

**Comment by brollb** [comment]
Additional statistics that could be interesting:
- RPC usage
- # of messages sent (maybe include the sender/receivers)

**Comment by gsteinLTU** [comment]
Yeah, most used RPCs would be very interesting to be able to visualize.
